### PR TITLE
[CHORE] Move Add Token Function to Metamask.ts file

### PR DESCRIPTION
**Pull request by tiffanydys**
# Description
- I've noticed that the `addTokenToMetamask` function is placed inside `Withdrawn.tsx` file
- This PR moves the `addTokenToMetamask` along with the other metamask functions found in `metamask.ts` file - for better tracking, reusability and consistency
- This also adds testnet SFL token data to help developers track the SFL token in testnet

## Type of change
- [x] [CHORE] (non-breaking change which fixes an issue)

# How Has This Been Tested?
`yarn dev`

https://user-images.githubusercontent.com/18549210/170470490-633efe19-c050-430b-83dc-38c1fe42deef.mov

`yarn test`

Screen Shot 2022-05-26 at 6 29 25 PM

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by adamhannigan — APPROVED** [review]
Much cleaner, well done 👏

File: src/lib/config.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ const NETWORK = import.meta.env.VITE_NETWORK as "mainnet" | "mumbai";
 const DONATION_ADDRESS = import.meta.env.VITE_DONATION_ADDRESS;
 
 const POLYGON_CHAIN_ID = NETWORK === "mainnet" ? 137 : 80001;
+const SFL_ADDRESS =
+  NETWORK === "mainnet"
+    ? "0xD1f9c58e33933a993A3891F8acFe05a68E1afC05"
```
Comment (adamhannigan) [inline src/lib/config.ts:7]: @tiffanydys 

Is this the same as `TOKEN_CONTRACT`?

Could we use the `TOKEN_CONTRACT` variable which injects either the testnet or mainnet address for the SFL token?

Comment (tiffanydys) [reply src/lib/config.ts]: Whoops sorry, didn't see that. Edited it out and made sure it still works:

Screen Shot 2022-05-30 at 8 09 53 AM

**Comment by jifenggg** [comment]
哔哔哔...→_→ 收到

**Pull request merged**